config.json

```diff
@@ -115,6 +115,17 @@
       ]
     },
     {
+      "uuid": "40d4bb98-2e4a-4579-810e-g1230bc158e9",
+      "slug": "complex-numbers",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "Mathematics",
+        "Number-theory"
```

Comment:
will any other exercise have this topic? if so, I'd like to see it added to them as well. If not, I'm not finding it particularly useful